coral-component-actionbar/examples/index.html

```diff
@@ -31,12 +31,12 @@
       });
     </script>
 
-    <link rel="stylesheet" href="../dist/css/coral.css">
```

Comment:
Why is this change necessary ? 
It seems to be that coral.css is downloaded correctly here
https://opensource.adobe.com/coral-spectrum/examples/#actionbar

Reply:
Reverted